core/src/main/java/org/apache/gravitino/storage/relational/service/RoleMetaService.java

```diff
@@ -237,6 +239,26 @@ private List<SecurableObjectPO> listSecurableObjectsByRoleId(Long roleId) {
         SecurableObjectMapper.class, mapper -> mapper.listSecurableObjectsByRoleId(roleId));
   }
 
+  public List<RoleEntity> listRolesByNamespace(Namespace namespace, boolean allFields) {
+    AuthorizationUtils.checkRoleNamespace(namespace);
+    String metalakeName = namespace.level(0);
+
+    if (allFields) {
+      throw new IllegalArgumentException("Don't support list all the fields of roles");
+    }
```

Comment:
Can we remove this `allFields` param in the `public List<RoleEntity> listRolesByNamespace(Namespace namespace, boolean allFields)` ?

Reply:
OK.